.github/workflows/build-and-test.yml

```diff
@@ -82,11 +86,10 @@ jobs:
         uses: actions/checkout@v2
       - uses: actions/setup-python@v1
       - name: Install OpenMPI, CMake, Boost library, Eigen and pkg-config
-        user: root
         run: |
-          apt-get -yy update
-          apt-get install -y libopenmpi-dev cmake libboost-all-dev libeigen3-dev pkg-config
-          rm -rf /var/lib/apt/lists/*
+          sudo apt-get -yy update
+          sudo apt-get install -y libopenmpi-dev cmake libboost-all-dev libeigen3-dev pkg-config
+          sudo rm -rf /var/lib/apt/lists/*
```

Comment:
Why `sudo` here and `su root` above?

Reply:
Removed the `su root` above as one is already root in the `precice:precice/develop` container. Here we need `root` because we are in a general `ubuntu:latest` container.